tests/test_client_functional.py

```diff
@@ -38,7 +38,7 @@ def handler(request):
     resp2 = yield from client.get('/')
     yield from resp2.read()
 
-    assert 1 == len(client._session.connector._conns)
+    assert 0 == len(client._session.connector._conns)
```

Comment:
Why do you change the test?

It looks like code disables keepalive somehow, I suspect a regression here.